tests/west/runner/test_nrfjprog.py

```diff
@@ -0,0 +1,239 @@
+# Copyright (c) 2018 Foundries.io
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+from unittest.mock import patch, call
+
+import pytest
+
+from west.runner.nrfjprog import NrfJprogBinaryRunner
+from .conftest import RC_KERNEL_HEX
+
+
+#
+# Test values
+#
+
+TEST_DEF_SNR = 'test-default-serial-number'  # for mocking user input
+TEST_OVR_SNR = 'test-override-serial-number'
+
+#
+# Expected results.
+#
+# These record lists of system calls made by the nrfjprog runner. Each
+# system call is a list of arguments passed to the check_call() method.
```

Comment:
Had to stare at this for a while to realize that it's testing that those particular commands are executed. Maybe the variable could be renamed to something like `EXPECTED_COMMANDS`, and the comment rewritten to something like this:

```
This dictionary maps different configurations to the commands we expect to be
executed for them. Verification is done by patching the check_call() method,
which is used to run the commands.
```